src/cdk-experimental/clipboard/copy-to-clipboard.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, EventEmitter, Input, Output} from '@angular/core';
+
+import {Clipboard} from './clipboard';
+
+/**
+ * Provides behavior for a button that when clicked copies content into user's
+ * clipboard.
+ *
+ * Example usage:
+ *
+ * `<button copyToClipboard="Content to be copied">Copy me!</button>`
+ */
+@Directive({
+  selector: '[copyToClipboard]',
```

Comment:
This just needs the `cdk` prefix here, so `cdkCopyToClipboard` (same for the class name, `CdkCopyToClipboard`)

Reply:
Done.